core/pom.xml

```diff
@@ -822,16 +822,6 @@
 			<artifactId>junit-jupiter</artifactId>
 			<scope>test</scope>
 		</dependency>
-		<dependency>
-			<groupId>org.junit.jupiter</groupId>
-			<artifactId>junit-jupiter-params</artifactId>
-			<scope>test</scope>
-		</dependency>
-		<dependency>
-			<groupId>org.mockito</groupId>
-			<artifactId>mockito-junit-jupiter</artifactId>
-			<scope>test</scope>
-		</dependency>
```

Comment:
Was dit toch niet nodig? Ik snap hem even niet

Reply:
Voor deze change niet nee, de mocks zullen we wel nodig gaan hebben voor de HttpSenderAuthenticationTest

Reply:
Deze zijn meegegaan in een vorig PR, maar voor nu zijn ze nog niet nodig.
Als we straks de tests met de mocks aan gaan passen zijn ze weer wel nodig

Reply:
En JUnit-Jupiter-Params zal ook weer nodig zijn (het verbaast me dat deze nu niet nodig is).
Daarom zou ik ze zelf liever erin laten. Maar ja.